js_runner.go

```diff
@@ -13,8 +13,8 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-
 	"github.com/robertkrimen/otto"
+	"os"
```

Comment:
We should group stdlib imports together (I think there's an goimports/gofmt IDE option for this)